.storybook/stories/KvFormKitchenSink.stories.js

```diff
@@ -166,40 +166,6 @@ export const KitchenSink = () => ({
 
 				</fieldset>
 
-				<fieldset>
```

Comment:
Sneaking this in as a cleanup, since going forward we'll just being using `<kv-dropdown>` rather than `<select>`